src/main/java/core/basesyntax/strategy/operation/SupplyOperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy.operation;
+
+public class SupplyOperationHandler implements OperationHandler {
```

Comment:
According to the checklist, the class should not begin with an empty line. Please remove the redundant empty line at the beginning of the class definition.

Reply:
where is emtpty line which must removed
?